yapapi/engine.py

```diff
@@ -188,14 +188,14 @@ async def create_demand_builder(
         return builder
 
     @property
-    def driver(self) -> str:
+    def payment_driver(self) -> str:
```

Comment:
same for the properties -> they also should still stay and be marked as deprecated

Reply:
Properties of the `Engine`? Public interface is now defined on `Golem` only.

Reply:
again... @zakaprov renamed them on Golem too ...